.github/workflows/release_note.py

```diff
@@ -35,7 +35,7 @@
 
 # https://github.com/gardenlinux/gardenlinux/issues/3044
 # Empty string is the 'legacy' variant with traditional root fs and still needed/supported
-image_variants = ['']
+image_variants = ['', '_usi', '_tpm2_trustedboot']
```

Comment:
I think `image_variants` is not used anymore. Am I missing something?

The variable, the comment above and its usage can be deleted if I'm not mistaken.